api/test/integration/test_rbac_white_agent_endpoints.tavern.yaml

```diff
@@ -1658,6 +1659,32 @@ stages:
           total_affected_items: 0
           total_failed_items: 1
 
+  # Agent assignment below requires the agent not to be in the group to succeed
+  - name: Remove agent 008 from group default (Allowed)
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/agents/group"
+      method: DELETE
+      headers:
+        Authorization: "Bearer {test_login_token}"
+      params:
+        agents_list: '008'
+        group_id: 'default'
+    response:
+      verify_response_with:
+        function: tavern_utils:healthcheck_agent_restart
+        extra_kwargs:
+          agents_list: ["008"]
+      status_code: 200
+      json:
+        error: 0
+        data:
+          affected_items:
+            - '008'
+          failed_items: []
+          total_affected_items: 1
+          total_failed_items: 0
+
```

Comment:
Perhaps not the most efficient solution but it was the only one that worked, tried modifying previous tests to change the state of the cluster when it reached the failed test but every tiny change caused other tests to fail. This was the simplest way to solve the issue.